src/cinder/Triangulate.cpp

```diff
@@ -96,34 +104,72 @@ void Triangulator::addPath( const Path2d &path, float approximationScale )
 void Triangulator::addPolyLine( const PolyLine2f &polyLine )
 {
 	if( polyLine.size() > 0 )
-		tessAddContour( mTess.get(), 2, &polyLine.getPoints()[0], sizeof(float) * 2, (int)polyLine.size() );
+		tessAddContour( mTess.get(), 2, polyLine.getPoints().data(), sizeof(vec2), (int)polyLine.size() );
 }
 
 void Triangulator::addPolyLine( const vec2 *points, size_t numPoints )
 {
 	if( numPoints > 0 )
-		tessAddContour( mTess.get(), 2, &points, sizeof(vec2), (int)numPoints );
+		tessAddContour( mTess.get(), 2, points, sizeof(vec2), (int)numPoints );
+}
+
+void Triangulator::addPolyLine( const vec3 *points, size_t numPoints )
+{
+	if( numPoints > 0 )
+		tessAddContour( mTess.get(), 3, points, sizeof(vec3), (int)numPoints );
 }
 
 TriMesh Triangulator::calcMesh( Winding winding )
 {
 	TriMesh result( TriMesh::Format().positions( 2 ) );
-	
-	tessTesselate( mTess.get(), (int)winding, TESS_POLYGONS, 3, 2, 0 );
+
+	float normal[3] = { 0, 0, 1 };
```

Comment:
So I'd started passing this in to avoid computing it, but I need to test it out. I think the old behavior would flip the normal to use a (0,0,-1) if the winding order was backwards. I need to add test cases for that.